### PR TITLE
Infrastructure/4804 update meta capabilities

### PR DESCRIPTION
**Pull request by hussain-t**
## Summary

Addresses issue:

- #4804 

## PR Author Checklist

- [x] My code is tested and passes existing unit tests.
- [ ] My code has an appropriate set of unit tests which all pass.
- [ ] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [ ] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Code Reviewer Checklist

- [x] Run the code.
- [x] Ensure the acceptance criteria are satisfied.
- [x] Reassess the implementation with the IB.
- [x] Ensure no unrelated changes are included.
- [x] Ensure CI checks pass.
- [ ] Check Storybook where applicable.
- [x] Ensure there is a QA Brief.

## Merge Reviewer Checklist

- [x] Ensure the PR has the correct target branch.
- [x] Double-check that the PR is okay to be merged.
- [x] Ensure the corresponding issue has a ZenHub release assigned.
- [x] Add a changelog message to the issue.

### CODE REVIEW

**Review by techanvil — CHANGES_REQUESTED** [review]
Thanks @hussain-t, a few minor comments but also a more involved one about the JSON encoding for you to take a look at.

File: assets/js/googlesitekit/datastore/user/permissions.js [hunk 1]
```diff
@@ -177,11 +177,21 @@ export const selectors = {
...
+	 * @param {Array}  args       List of rest of the arguments. Specifically one or many module slugs.
 	 * @return {(boolean|undefined)} TRUE if the current user has this capability, otherwise FALSE. If capabilities ain't loaded yet, returns undefined.
 	 */
 	hasCapability: createRegistrySelector(
-		( select ) => ( state, capability ) => {
+		( select ) => ( state, capability, ...args ) => {
 			const capabilities = select( CORE_USER ).getCapabilities();
+
+			if ( args.length > 0 ) {
+				// The backend returns the meta capabilities' slug within `single-quotes'.
```
[the remark this answers is not in the document]
Comment (hussain-t) [reply assets/js/googlesitekit/datastore/user/permissions.js:188]: Thanks @techanvil that's a good catch 👍

**Review by techanvil — APPROVED** [review]
Thanks @hussain-t, this LGTM. There was just one super trivial comment about spacing consistency that you missed, literally just adding an empty line, so I have made this change and approved. Cheers

**Comment by hussain-t** [comment]
Thanks @techanvil, sorry I missed that :)

**Comment by techanvil** [comment]
> Thanks @techanvil, sorry I missed that :)

No worries, cheers Hussain!

**Pull request merged**